cloud/src/meta-service/meta_service.cpp

```diff
@@ -2432,26 +2455,10 @@ static bool put_delete_bitmap_update_lock_key(MetaServiceCode& code, std::string
 void MetaServiceImpl::get_delete_bitmap_update_lock_v2(
         google::protobuf::RpcController* controller,
         const GetDeleteBitmapUpdateLockRequest* request,
-        GetDeleteBitmapUpdateLockResponse* response, ::google::protobuf::Closure* done) {
+        GetDeleteBitmapUpdateLockResponse* response, ::google::protobuf::Closure* done,
+        std::string& instance_id, MetaServiceCode& code, std::string& msg, std::stringstream& ss) {
```

Comment:
the new added parameters seems not used?

Reply:
parameters ss,code,msg,instance_id are created in RPC_PREPROCESS(get_delete_bitmap_update_lock); now i move this line into function get_delete_bitmap_update_lock and get_delete_bitmap_update_lock_v1,get_delete_bitmap_update_lock_v2 will use these parameters